src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/object.expected.yml

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
extra unneeded line

Reply:
Every expected file has it

https://github.com/aurimasniekis/symfony/blob/be54d774ea5fd4cf9496fb4b39fc81c9ddefc25f/src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/basic.expected.yml#L1